src/navigation/components/SocialIcon/SocialIcon.tsx

```diff
@@ -13,15 +13,15 @@ export interface NavigationSocialItemOwnProps extends NavigationSocialItem {
     className?: string;
 }
 
-const SocialIcon: React.FC<NavigationSocialItemOwnProps> = ({icon, url, className}) => {
+const SocialIcon: React.FC<NavigationSocialItemOwnProps> = ({icon, url, className, urlTitle}) => {
     const iconData = getMediaImage(icon);
     const {alt} = iconData;
 
     return (
         <a
             href={url}
             aria-label={alt}
-            title={alt}
+            title={urlTitle || alt}
```

Comment:
Can `alt` of icon be the title of a link?

Reply:
Yes, but the accessible name will read:
1) link text
2) image alt
3) link title

Is icon's `alt` is the same as the link title,  the screenreader will read two sentences twice, it is not really good. So I think they should differ